source/user-manual/reference/ossec-conf/remote.rst

```diff
@@ -143,6 +144,26 @@ Sets the time to close the RIDS files for agents that don't report new events in
 | **Allowed values** | A positive number that should contain a suffix character indicating a time unit, such as, s (seconds), m (minutes), h (hours), d (days). |
 +--------------------+------------------------------------------------------------------------------------------------------------------------------------------+
 
+connection_overtake_time
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 4.5.2
+
+Sets the time to wait before considering a TCP connection down. A value of 0 disables this assessment of connection activity.
```

Comment:
```suggestion
Sets the time to wait before considering a connection with a TCP client down. A value of 0 disables this assessment of connection activity.
```

Reply:
Done! I only  add when this setting will work.